README.md

```diff
@@ -53,28 +77,24 @@ After following the steps, run the command below and follow the instructions to
 ts-node src/google/create-token.ts
 ```
 
-## Deployment
+## Development
 
-Before deployment, create an `.env` file using the fields mentioned below:
+To add new features to the bot. Fork the repository and add your feature to `src/meetbot/features`. After finishing your changes, [run the bot locally](#running-the-bot-locally) to check if your feature loads and test your changes. 
 
-```
-GOOGLE_PASSWORD=              # Password for your bot/user that is used to join Google Meet. 
-GOOGLE_TOTP_SECRET=           # [Optional] If the bot account has 2FA auth, then add 2FA secret here.
-GOOGLE_LOGIN=                 # Email-ID/username of your bot/user
-TOKEN_PATH=token.json         # [Default] Path to token file 
-HTTP_PORT=80                  # Port to run express server on. For prod, use port 80.
-RUNNING_ALPINE=0              # [Default] If base image is alpine then set the env var, otherwise don't. 
-```
+To develop the UI, run `npm run dev-ui`. This will spin up the development server. You should then be able to access the dashboard at `http://localhost:3000/`. Note that the data you will see on the tables are actually being mocked by MirageJS, and not actual request is going to the API. This makes it easier to develop both components in an entirely decoupled way.
```

Comment:
```suggestion
To develop the UI, run `npm run dev-ui`. This will spin up the development server. You should then be able to access the dashboard at `http://localhost:3000/`. Note that the data you will see on the tables is only mocked and no actual requests are sent to the API. This makes it easier to develop the UI independently.
```